### DISCUSSION TITLE
Authentication needed for /health endpoint?

### DISCUSSION THREAD

**Discussion by EugenMayer** [cantaloupe-project/cantaloupe#520] [General]
I;am trying to utilize the health endpoint, but it returns 403. Any way to disable that?
I'am running the server in the private network and health will not be exposed to the outer edge router - i would need that healh for consul service checks and so on.

Thanks!

**Comment by adolski** [comment]
There is a quirk in 5.x whereby the `endpoint.api.enabled` config key must be set to `true` to enable the health endpoint.

In 6.x there is a new `endpoint.health.enabled` config key that will fix that.
